src/Module.Server/Balancing/GameMatch.cs

```diff
@@ -7,6 +9,13 @@ internal class GameMatch
     internal List<WeightedCrpgUser> Waiting { get; set; } = new();
 }
 
+internal class DTVGameMatch
+{
+    internal List<CrpgUser> TeamA { get; set; } = new();
+    internal List<CrpgUser> TeamB { get; set; } = new();
+    internal List<CrpgUser> Waiting { get; set; } = new();
+}
+
```

Comment:
Gamematch are used solely as an interface between game and the balancer. 
Can you explain How there would be two team of players in dtv?
If not , i do not think you need a balancer.